src/db/migrate.rs

```diff
@@ -29,15 +28,23 @@ macro_rules! migration {
         }
         impl PostgresMigration for Amigration {
             fn up(&self, transaction: &mut Transaction) -> Result<(), PostgresError> {
-                info!(
+                #[cfg(not(test))]
+                use log::info as log_level;
+                #[cfg(test)]
+                use log::trace as log_level;
+                log_level!(
```

Comment:
```suggestion
                log!(
                    if cfg!(test) { Level::Trace } else { Level::Info },
```

+ appropriate imports, seems a little cleaner to me.